frontend/src/pages/modelRegistrySettings/CreateModal.tsx

```diff
@@ -206,7 +206,7 @@ const CreateModal: React.FC<CreateModalProps> = ({ onClose, refresh, modelRegist
         kind: 'ModelRegistry',
         metadata: {
           name: nameDesc.k8sName.value || translateDisplayNameForK8s(nameDesc.name),
-          namespace: modelRegistryNamespace!,
+          namespace: modelRegistryNamespace || '',
```

Comment:
@ikeola13 we should avoid falling back to `''` wherever possible. Instead can you put an early return at the top of this function if modelRegistryNamespace isn't defined? If we don't have it we can't submit.